tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/cdi/OpenTelemetryBeanTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016-2023 Contributors to the Eclipse Foundation
+ *
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.eclipse.microprofile.telemetry.tracing.tck.cdi;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import jakarta.inject.Inject;
+
+class OpenTelemetryBeanTest extends Arquillian {
+
+    private static final String SPAN_NAME = "MySpanName";
+    private static final String INVALID_SPAN_ID = "0000000000000000";
+
+    @Deployment
+    public static WebArchive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class)
+                .addAsResource(new StringAsset("otel.sdk.disabled=false"),
+                        "META-INF/microprofile-config.properties");
+    }
+
+    @Inject
+    OpenTelemetry openTelemetry;
```

Comment:
The test doesn't specify any CDI bean or descriptor, therefore CDI doesn't get enabled in Payara. I recommend adding a `beans.xml` file in this case or any scope annotated bean into the deployment

Reply:
Thank you @pdudits - I have added a `beans.xml`